### PR TITLE
[documentation] How to debug connection to proxy cannot be established

### PR DESCRIPTION
**Pull request by violetagg**

### CODE REVIEW

File: docs/asciidoc/tcp-client.adoc [hunk 1]
```diff
@@ -310,6 +310,12 @@ include::{examplesdir}/proxy/Application.java[lines=18..38]
 ----
 ====
 
+NOTE: Netty's HTTP proxy support always uses `CONNECT` method in order to establish a tunnel to the specified proxy regardless of the scheme that is used `http` or `https`.
```
Comment (simonbasle) [inline docs/asciidoc/tcp-client.adoc:313]: unless I missed something this is a copy-paste of the faq entry. consider linking to the faq entry or using a [tagged region](https://docs.asciidoctor.org/asciidoc/latest/directives/include-tagged-regions/) to directly import the relevant part(s) from the `faq.adoc` source

**Review by simonbasle — CHANGES_REQUESTED** [review]
content looks good. made suggestion to mutualize and organize the content using asciidoc include directive

File: docs/asciidoc/index.asciidoc [hunk 2]
```diff
@@ -55,3 +55,10 @@ https://github.com/reactor/reactor-netty/edit/main/docs/asciidoc/udp-client.adoc
 to "<<udp-client>>"
 endif::[]
 
+[appendix]
```
[the remark this answers is not in the document]
Comment (violetagg) [reply docs/asciidoc/index.asciidoc:58]: > shouldn't the appendix/faq be linked first and foremost at the end of index.adoc?

Reactor Netty has only `index.asciidoc`, it doesn't have `index.adoc`. I don't understand.

> a link or include of the <<faq.proxy-connect-method>> entry can then be included in both udp-client.adoc

There is no section for proxy for UDP client, I'll add it

File: docs/asciidoc/tcp-client.adoc (hunk 1 above)
Comment (violetagg) [reply docs/asciidoc/tcp-client.adoc:313]: ok

File: docs/asciidoc/index.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (simonbasle) [reply docs/asciidoc/index.asciidoc:58]: ah my bad, I misread the diff and thought that the below was the edited file 🤦 
```
@@ -55,3 +55,10 @@ https://github.com/reactor/reactor-netty/edit/main/docs/asciidoc/udp-client.adoc
```

**Review by simonbasle — APPROVED** [review]

**Comment by violetagg** [comment]
@simonbasle Thanks for the review!

**Pull request merged**